plugins/bdv/bdv_funcs.c

```diff
@@ -0,0 +1,78 @@
+/* bdv_funcs.c */
```

Comment:
Are the BDV files intended to be part of this patch or are they left over from your local testing?